src/Command/WelcomeCommand.php

```diff
@@ -40,15 +43,25 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $this->outputImage($io);
 
-        $io->info('Welcome to your new Bolt project. To set up the database, run `bin/console bolt:setup` inside your new project folder.');
+        $io->info('Welcome to your new Bolt project.');
         $io->text('For the full setup instructions, and other documentation, visit <href=https://docs.bolt.cm/installation/installation>https://docs.bolt.cm/installation/installation</>');
         $io->text('To ask questions and learn from our community, join our Slack channel: <href=https://slack.bolt.cm/>https://slack.bolt.cm/</>');
         $io->text('Additional resources and tips are available at <href=https://bolt.tips/>https://bolt.tips/</>');
 
-        $io->note('Do you develop websites with Bolt? You can join the growing list of fellow Bolters at people.boltcms.io, open source and free.');
+        $io->note('If you wish to continue with SQLite, you can answer \'Y\' to the next question. If you\'d like to use MySQL or PostgreSQL, answer \'N\', configure `.env.local`, and then continue the setup.');
```

Comment:
Nice, should we keep the uppercase and/or lowercase consistent? Here it is `N`, in the question below it is lowercase `n`.

Reply:
Fixed!